test/functionalTest/cases/1582_alarmManager/alarm_manager_bad_input.test

```diff
@@ -175,9 +175,9 @@ Date: REGEX(.*)
 
 07. grep in log-file
 ====================
-Raising alarm BadInput 0.0.0.0: service '/v2/noService' not found
-Releasing alarm BadInput 0.0.0.0
-Raising alarm BadInput 0.0.0.0: bad verb for url '/v2/entities', method 'PROUST'
+Raising alarm BadInput REGEX((0.0.0.0|127.0.0.1)): service '/v2/noService' not found
```

Comment:
If we want to fix this in the future, please create an issue about it (techdebt nice2have maybe?), including in its description all the knowledgue we have about it (we were having discussion about this at skype, including some links we found about this topic... it would be great to include all the relevant information in the issue for not lossing it).

Reply:
All I could remember:  https://github.com/telefonicaid/fiware-orion/issues/2983